tests/contrib/langchain/test_langchain_llmobs.py

```diff
@@ -517,6 +585,60 @@ def test_llmobs_anthropic_chat_model(self, langchain_anthropic, mock_llmobs_span
         assert mock_llmobs_span_writer.enqueue.call_count == 1
         _assert_expected_llmobs_llm_span(span, mock_llmobs_span_writer, input_role="user")
 
+    @pytest.mark.skipif(sys.version_info < (3, 10, 0), reason="Requires unnecessary cassette file for Python 3.9")
+    def test_llmobs_embedding_query(self, langchain_community, mock_llmobs_span_writer, mock_tracer):
+        if langchain_community is None:
+            pytest.skip("langchain_community not installed which is required for this test.")
+        embedding_model = langchain_community.embeddings.OpenAIEmbeddings()
```

Comment:
```suggestion
    def test_llmobs_embedding_query(self, langchain_openai, mock_llmobs_span_writer, mock_tracer):
        if langchain_openai is None:
            pytest.skip("langchain_openai not installed which is required for this test.")
        embedding_model = langchain_openai.OpenAIEmbeddings()
```
Can we use the `langchain_openai` package instead? This is more up-to-date than `langchain_community`.